pilot/cmd/pilot-agent/status/dialer_windows.go

```diff
@@ -0,0 +1,41 @@
+//go:build windows
+// +build windows
```

Comment:
Compiling istioctl on windows will fail without this file because it imports `pilot/cmd/pilot-agent/status`
https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/43021/release-test_istio/1618941069653708800
```
GOOS=windows LDFLAGS='-extldflags -static -s -w' common/scripts/gobuild.sh /tmp/tmp.3iERGbbxHP/build/work/src/istio.io/istio/out/linux_amd64/release/istioctl-win.exe ./istioctl/cmd/istioctl
go: downloading github.com/inconshreveable/mousetrap v1.1.0
go: downloading github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1
# istio.io/istio/pilot/cmd/pilot-agent/status
pilot/cmd/pilot-agent/status/server.go:908:34: cannot use int(fd) (value of type int) as type syscall.Handle in argument to syscall.SetsockoptLinger
```